src/zope/component/tests/test__declaration.py

```diff
@@ -80,22 +80,11 @@ def _run_generated_code(self, code, globs, locs,
                             fails_under_py3k=True,
                            ):
         import warnings
-        #from zope.component._compat import PYTHON3
-        PYTHON3 = False
```

Comment:
OK.  I still think we should be deprecating `adapts` on the `There should be one-- and preferably only one --obvious way to do it.` principle.